### PR TITLE
Failover content for Telephony Inline Hook

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** added failover information, a link to the troubleshoot section in the API content, and an example JSON response with the discussed properties
- **Is this PR related to a Monolith release?** No

Netlify link: https://preview-4956--reverent-murdock-829d24.netlify.app/docs/guides/telephony-inline-hook/nodejs/main/#failover-to-okta-telephony-providers

### Resolves:

* [OKTA-748316](https://oktainc.atlassian.net/browse/OKTA-748316)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/96cc9b48da5c57a5e707dc7ad8b7e9f44a1846ae/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/37b7cc87-d358-4c7f-a9ba-74704163de52) | 92  | 73  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.6cb5d701-5e70-4760-92f2-c33de0303e65?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 1]
```diff
@@ -144,10 +144,38 @@ The following code is used to send the SMS or voice call to the user:
...
+    }
+  ]
+}
+```
+
+### Failover to Okta telephony providers
+
+There are several common causes of failure for telephony inline hooks. When a response fails, Okta attempts to send the OTP using the Okta telephony providers. However, this failover is heavily rate-limited.
+
+See the [Troubleshoot](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/InlineHook/#tag/InlineHook/operation/createTelephonyInlineHook) section in the Telephony Inline Hook API reference for a list that includes the impact of the issue and error visibility.
```
Comment (diptidesai-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:177]: trying to understand - what issue are we referring to?

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/20a4d3925c62643b5231fe05bc463ac8a56700f5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/32138255-9d38-4b18-83ea-009373a3784d) | 92  | 73  | 78  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a95dd163-7615-41a2-982d-1e04cc1c037a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/3e7cd60610a71633b04299ced82d2e94d05ab112/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0fd6f57e-91e6-4f8d-ac78-1adaf8612c5f) | 93  | 74  | 96  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.7ff13a50-f272-4ae4-809e-eb2807a198bd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/1be1e43c5f05c6e40c32dadd33059f07fd770b35/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/24e3d49d-b501-482e-a476-befb77bae19d) | 93  | 72  | 96  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.99fdef99-4fea-47f7-9452-7b8b7a56417b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/3a57e8ac4f3fc88408915fea700c0145034a7d2a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dbdd8714-9f3d-40c8-9e08-9bfe6a3a0418) | 93  | 72  | 96  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.fae87bda-5424-4832-9a18-d8c69bdc14ed?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/dd290914a7d68541a8787cb187100b756161d3ee/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d5c60893-8f6d-4e3e-86ab-ae6af653dde0) | 93  | 72  | 96  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9f9dae48-91df-4e0c-a4dc-e62576eac2e1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by diptidesai-okta — APPROVED** [review]
Revisions LGTM. Thanks

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ This guide provides an example of an Okta telephony inline hook. This guide uses
 
 ## About telephony inline hook implementation
 
-The Okta telephony inline hook allows you to integrate your own custom code into Okta flows that send SMS or voice call messages (except Okta Verify enrollment). You can integrate this hook with enrollment, authentication, and recovery flows that involve phone authenticators. Okta uses your external provider to deliver the one-time passcode (OTP) to the Requester. The provider can respond with commands that indicate if the delivery was successful.
+The Okta telephony inline hook enables you to integrate your own custom code into Okta flows that send SMS or voice call messages (except Okta Verify enrollment). You can integrate this hook with enrollment, authentication, and recovery flows that involve phone authenticators. Okta uses your external provider to deliver the one-time passcode (OTP) to the Requester. The provider can respond with commands that indicate if the delivery was successful.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:48]: Does requester need to be capitalized?

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Verify that your org has the Phone authenticator added and enabled for **Authent
 1. Verify that the **Authentication and recovery** option is selected.
 1. Click **Save** if made any changes.
 
-> **Note:** If a Phone authenticator isn't already added, click **Add Authenticator** and then click **Add** on the Phone tile and make sure that you select the options mentioned earlier and click **Add**.
+> **Note:** If a Phone authenticator isn't already added, click **Add Authenticator**, and then **Add** on the Phone tile. Make sure that you select the options mentioned earlier, and then click **Add**.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:73]:
```suggestion
> **Note:** If a phone authenticator isn't already added, click **Add Authenticator**, and then **Add** on the Phone tile. Make sure that you select the options mentioned earlier, and then click **Add**.
```

File: packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md (hunk 2 above)
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md:48]:
```suggestion
The Okta telephony inline hook enables you to integrate your own custom code into Okta flows that send SMS or voice call messages (except Okta Verify enrollment). You can integrate this hook with enrollment, authentication, and recovery flows that involve phone authenticators. Okta uses your external provider to deliver the one-time passcode (OTP) to the requester. The provider can respond with commands that indicate if the delivery was successful.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/30fd14a55c300ae8887b8588b706df75594a8372/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2573821e-95e7-49ed-91bf-cf4811bef414) | 90  | 72  | 96  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.ac519471-7735-47c6-b92a-d5b717a1daee?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/ea7cdb06e6f691f8f1caf2b66c5ae4009942e042/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b924b75e-a38d-4e3d-967f-2bf76e561b6a) | 90  | 72  | 96  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.2695c148-2ed9-4e24-89cf-feed2b369c44?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**